velox/common/memory/ByteStream.h

```diff
@@ -281,12 +254,6 @@ class ByteStream {
   /// the last range.
   size_t size() const;
 
-  /// Returns the remaining size left from current reading position.
-  size_t remainingSize() const;
```

Comment:
i still see this being used in PrestoSerializer.cpp in the main branch, are we removing those in a different PR?
https://github.com/facebookincubator/velox/blob/40f9dc31165fc98d681a7814221b6b5242b15cef/velox/serializers/PrestoSerializer.cpp#L54C22-L54C28

Reply:
@bikramSingh91 I removed these in https://github.com/prestodb/presto/pull/21380